internal/commands/push/command.go

```diff
@@ -179,6 +214,46 @@ func (cmd *Command) Handler(profile *cli.Profile, ui terminal.UI, clients cli.Cl
 		ui.Print(terminal.NewTextLog("Uploaded dependencies archive"))
 	}
 
+	if cmd.inputs.IncludeHosting {
+		s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+		s.Suffix = " Importing hosting assets..."
+
+		importHosting := func() error {
+			s.Start()
+			defer s.Stop()
+
+			return hosting.UploadHostingAssets(
+				clients.Realm,
+				to.GroupID,
+				to.AppID,
+				hostingDiffs,
+				func(err error) { ui.Print(terminal.NewWarningLog(err.Error())) },
+			)
+		}
+
+		if err := importHosting(); err != nil {
+			return err
+		}
+		ui.Print(terminal.NewTextLog("Import hosting assets"))
+
+		if cmd.inputs.ResetCDNCache {
+			s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+			s.Suffix = " Resetting CDN cache..."
```

Comment:
thoughts on putting some context for the user on why this is necessary?

Reply:
i could probably use that same context 😬 

I kept it short/naive here b/c the user has to explicitly include the flag to perform this action anyways... so theoretically they should have an idea/know what theyre doing

open to suggestions to add on here if you have them though